docs/plugins/exatrkx.md

```diff
@@ -1 +1,49 @@
-# ExaTrkX
+(exatrkxplugin)=
+# Exa.TrkX plugin
+
+This plugin contains a track finding module based on Graph Neural Networks (GNNs) which is developed by the [Exa.TrkX](https://exatrkx.github.io/) team. Build instructions and dependencies can be found in the [README](https://github.com/acts-project/acts/blob/main/Plugins/ExaTrkX/README.md) of the plugin.
+
+## Model
+
+The Exa.TrkX pipeline is a three-stage GNN-based algorithm:
+
+1) **Graph building** (*Embedding stage*): This is done via metric learning approach. A neural network tries to learn an mapping that minimizes the distance between points of the same track in the embedded space. In this embedded space is then a graph built using a fixed nearest-neighbor search.
+2) **Graph size reduction** (*Filter stage*): The resulting graph is typically to for processing with a graph neural network. Therefore a simple classifier reduces the graph size further.
+3) **Edge classification** (*GNN stage*): Finally, a GNN is used to find the edges in the Graph that belong to tracks. This is done by scoring each edge with a number between 0 and 1, and a corresponding *edge cut*.
+
+Finally, the track candidates must be extracted by a algorithm that finds the *weakly connected components* in the graph depending on the *edge cut*.
+
+## Implementation
+
+### Neural network backend
+
+Currently there are two backends available:
+
+- The **TorchScript** backend requires besides *libtorch* also *TorchScatter* that implements the scatter operators used in the graph neural network. The corresponding class is {class}`Acts::ExaTrkXTrackFindingTorch`.
+- The **ONNX** backend is currently not as well maintained as the TorchScript backend. The main reason is, that there are scatterin operators missing in ONNX, so that the graph neural network cannot be exported correctely in the `.onnx` format. The implementation also uses *cugraph* for graph building instead of *Boost.Graph*. The corresponding class is {class}`Acts::ExaTrkXTrackFindingOnnx`.
+
+### Graph building
+
+Both backends use currently different libraries for graph building.
+
+- The TorchScript backend uses *Boost.Graph* for graph building.
+- The ONNX backend uses the *cugraph* library for graph building
+
+## API and examples integration
+
+All implementations should inherit from {class}`Acts::ExaTrkXTrackFindingBase`, so that they can be passed via a `std::shared_ptr` to the {class}`ActsExamples::TrackFindingAlgorithmExaTrkX`. The algorithm interfaces to the algorithm is the `getTracks` function:
```

Comment:
"should"  or just "inherits". The justification seems too technical. I would suggest to just write that this interface is used by that algorithm.